Apromore-Clients/manager-client/src/main/java/org/apromore/manager/client/ManagerServiceImpl.java

```diff
@@ -110,10 +109,87 @@
 
     private boolean isGEDMatrixReady = true;
     
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
     private static final Logger LOGGER = LoggerFactory.getLogger(ManagerServiceImpl.class);
 
+    @Bean
+    public RestTemplate restTemplate() {
+        final SimpleClientHttpRequestFactory simpleClientHttpRequestFactory = new SimpleClientHttpRequestFactory();
+
+        simpleClientHttpRequestFactory.setConnectTimeout(HTTP_CONN_TIMEOUT_MILLS);
+        simpleClientHttpRequestFactory.setReadTimeout(HTTP_CONN_TIMEOUT_MILLS);
+
+        return new RestTemplate(simpleClientHttpRequestFactory);
+    }
+
+
+// Implementation of ManagerService
+
+    /**
+     * Logout, in Keycloak, user sessions associated with said user.
+     *
+     * @param username The username of the user to logout.
+     *
+     * @return <code>true</code> if the user was logged-out, <code>false</code> otherwise.
+     */
+    @Override
+    public boolean logoutUserAllSessions(final String username) throws Exception {
+        Boolean restRespResult = false;
+
+        final RestTemplate restTemplate = restTemplate();
+
+        final Properties securityMsProps = readSecurityMsProperties();
 
-    // Implementation of ManagerService
+        try {
+            restRespResult = restTemplate.getForObject(
+                            securityMsProps.getProperty(SECURITYMS_HTTP_LOGOUT_URL) + username,
+                            Boolean.class);
+            logger.debug("\n\nrestRespResult: {}", restRespResult);
+        } catch (final Exception e) {
```

Comment:
Huh.  I didn't know you could use `final` here!

Reply:
:) [DONE]